eng/scripts/Verify-Readme.ps1

```diff
@@ -0,0 +1,16 @@
+# Wrapper Script for Readme Verification
```

Comment:
The script here applies to other languages as well. Better to put it in eng/common and update other language to use the same script.